src/views/Advertisement/Advertisement.js

```diff
@@ -0,0 +1,40 @@
+import React, { PropTypes } from 'react'
+import cx from 'classnames'
+import {
+  customPropTypes,
+  getElementType,
+  META,
+} from '../../lib'
+
+/**
+*An ad displays third-party promotional content
+*/
+function Advertisement(props) {
+  const {
+    className,
+  } = props
+
+  const classes = cx(
+    'ui',
+    'ad',
+    className
+  )
+  const ElementType = getElementType(Advertisement, props)
+
+  return <ElementType className={classes} />
```

Comment:
What should be the `children` of this component? If I had the line below, about 11 tests fail
```
if (!_.isNil(children)) {
    return <ElementType {...rest} className={classes}>{children}</ElementType>
  }
```